### PR TITLE
teehee, tutor colors on ref book nav

### PR DESCRIPTION
**Pull request by pandafulmanda**
On ref book nav hover :).  What you think, @Fredasaurus rex?

small-thing

### CODE REVIEW

**Comment by Fredasaurus** [comment]
Win

File: resources/styles/components/reference-book/navbar.less [hunk 1]
```diff
@@ -47,7 +72,17 @@
...
       border-right: 1px solid @tutor-neutral-light;
       background: @tutor-neutral-lightest;
-      &:before{ content: @fa-var-outdent; }
+      &::before {
+        content: @fa-var-outdent;
+        .tutorize-nav(open);
+      }
+
+      &:hover {
+        &::before {
```
Comment (philschatz) [inline resources/styles/components/reference-book/navbar.less]: Could this use the `.clearfix()` mixin?

File: resources/styles/components/reference-book/navbar.less [hunk 2]
```diff
@@ -1,3 +1,18 @@
+.tutorize-nav(){
+  .x-icon-size(28px);
+  height: 24px;
+  background-position: 0 -1px;
+}
+
+.tutorize-nav(close){
+  .x-icon-bg('list-close');
```
Comment (philschatz) [inline resources/styles/components/reference-book/navbar.less]: Mind if I change this file to be `toc-close`?

Comment (pandafulmanda) [inline resources/styles/components/reference-book/navbar.less]: im not sure what you mean by this, but i don't mind

**Pull request merged**